xbmc/pvr/epg/EpgInfoTag.h

```diff
@@ -301,14 +301,25 @@ class CPVREpgInfoTag final : public ISerializable,
    * @brief Get the parental rating of this event.
    * @return The parental rating.
    */
-  int ParentalRating() const;
+  unsigned int ParentalRating() const;
 
   /*!
    * @brief Get the parental rating code of this event.
    * @return The parental rating code.
    */
-  const std::string& ParentalRatingCode() const { return m_strParentalRatingCode; }
+  const std::string& ParentalRatingCode() const;
 
+  /*!
+   * @brief Get the parental rating icon path of this event.
+   * @return Path to the parental rating icon.
+   */
+  const std::string& ParentalRatingIcon() const;
+
+  /*!
+   * @brief Get the parental rating source of this event.
+   * @return The parental rating source.
+   */
+  const std::string& ParentalRatingSource() const;
```

Comment:
Change these two methods to be inline, like `ParentalRationCode()`.

Reply:
Done.